src/views/menus/navigation-menu.styles.js

```diff
@@ -49,5 +49,21 @@ export default css`
   vaadin-details::part(toggle) {
     background: var(--color-sidebar-menu);
     color: var(--color-menu-items);
+    border-right: none;
+    border-bottom: none;
+    border-left: none;
+    font-family: var(--material-font-family);
+    font-size: var(--material-button-font-size);
+  }
+
+  vaadin-details::part(summary) {
+    border-top: 1px solid rgba(255, 255, 255, 0.4);
+  }
+
+  .file-list ul {
+    padding: 0 0 0 36px;
```

Comment:
This padding on the left makes it look very odd for some languages:
![image](https://user-images.githubusercontent.com/49606945/76200536-188b8c80-61f2-11ea-9be1-2aefc7b24bbd.png)
Note that most languages have a fairly long filename here. Only tibetan doesn't.

Reply:
This is still there so I will upload a small change to the padding to make it `padding: 0 12px 0 12px;` instead of `padding: 0 0 0 36px;`. That will make it much nicer.

Reply:
I have to take changes into the left menu, I will do it on a different branch later on